orbit/cmd/orbit/shell.go

```diff
@@ -98,6 +105,14 @@ var shellCommand = &cli.Command{
 			osquery.WithFlags([]string{"--database_path", osqueryDB}),
 		}
 
+		certPath := getCertPath(c)
+		if exists, err := file.Exists(certPath); err == nil && exists {
```

Comment:
Looks good!

One nit, please do a level.Error log when `err != nil` (so it doesn't silently fail)

Reply:
Apologies @lucasmrod an obvious omission on my part - should be fixed in 08a2090a57277651448505ff8f81f4cb258257c4